sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1422,8 +1424,10 @@ public static partial class SpecializedBlobExtensions
     {
         public static Azure.Storage.Blobs.Specialized.AppendBlobClient GetAppendBlobClient(this Azure.Storage.Blobs.BlobContainerClient client, string blobName) { throw null; }
         public static Azure.Storage.Blobs.Specialized.BlobBaseClient GetBlobBaseClient(this Azure.Storage.Blobs.BlobContainerClient client, string blobName) { throw null; }
+        public static Azure.Storage.Blobs.BlobContainerClient GetBlobContainerClient(this Azure.Storage.Blobs.Specialized.BlobBaseClient client) { throw null; }
```

Comment:
I almost wonder if we want the name `Parent` in there like `GetParentBlobContainerClient` or something to indicate you're moving up?  I could go either way since this is an advanced API.

Reply:
It was itching to do this, but wanted to wait and see if somebody speaks up. I'll make that change.